redwood-widget/src/jsMain/kotlin/app/cash/redwood/widget/RedwoodHTMLElementView.kt

```diff
@@ -67,6 +68,12 @@ private class RedwoodHTMLElementView(
       UiConfiguration(
         darkMode = colorSchemeQuery.matches,
         viewportSize = Size(width = element.offsetWidth.dp, height = element.offsetHeight.dp),
+        layoutDirection = when (element.dir) {
```

Comment:
Not sure this will work consistently. It seems like many HTML elements inherit `dir` without explicitly specifying it. But maybe it's OK to say Redwood elements need to specify `dir` if they want layout direction to work?

Reply:
Also there's a bunch of caveats listed on the [documentation page](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir).

Reply:
Yeah we can say you need to explicitly define it for now. No one is really using HTML seriously right now, and certainly not in a context where RTL would show up. But it's nice to not forget about it.